07/ID_tryExcept

```diff
@@ -0,0 +1,52 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+
+# MONTHS 30 DAYS
+months_30 = [4, 6, 9, 11]
+
+# MONTHS 31 DAYS
+months_31 = [1, 3, 5, 7, 8, 10, 12]
+
+# YEARS
+years = list(range(86, 100)) + list(range(00, 23))
+ID_only_numbers = []
+
+while True:
+
+    ID = input('Napiš rodné číslo: ')
+    if ID[6] == "/":
+        try:
+            a = int(ID[:6])
+            b = int(ID[7:])
+        except ValueError:
+            print("v rodném čísle jsou nepovolené znaky")
+            continue
+    else:
+        print("lomítky chybí nebo je na špatném místě")
+        continue
+
+    ID_only_numbers = ID[:6] + ID[7:]
+    modulo_eleven = int(ID_only_numbers)
+    year_ID = int(ID_only_numbers[:2])
+    month_ID = int(ID_only_numbers[2:4])
+    day_ID = int(ID_only_numbers[4:6])
+    end = int(ID_only_numbers[6:])
+
+    if (modulo_eleven % 11 == 0) and (year_ID in years) and (end < 6000):
+
+        if month_ID > 50:
+            month_ID = month_ID % 50
+
+        if month_ID == 2 and day_ID <= 29:
+            print('Zadané rodné číslo je v pořádku')
+            quit()
```

Comment:
Prosím quit() nie, používaj break. O tom prečo sa to nemá používať mimo konzolu som ti písala :)

Reply:
nebudu, slibuji :D 
a už vím, proč jsem ho tam dávala... když dám break, tak v konzoli ve výstupu se vypíše None a tomu jsem se chtěla vyhnout a právěkdyž jsem dala quit(), tak se to tam nevypíše